elastic/kibana
# [Uptime] Update label on SnapshotHistogram y-axis

## Summary

For 7.3.0 we have updated the _Snapshot Histogram_ component to show _all_ pings, rather than all monitors, because the old component did not stay consistent with the way our data has changed.

As a result, we should have updated the y-axis label on the visualization; we failed to do this correctly. This patch simply updates the label.

## Testing this PR
0. Start up ES/Kibana/Heartbeat with at least 1 monitor.
1. Load up Uptime UI and select a time range such that data appears in the bar series on the _Overview_ page.
  a. Ensure that you see the new `Pings` label
2. Drill into a monitor to load the _Monitor_ page.
  a. Ensure that you see the new `Pings` label

![image](https://user-images.githubusercontent.com/18429259/61142358-8b930680-a49d-11e9-8214-afaf337b86de.png)
